Sources/Views/MessageLabel.swift

```diff
@@ -306,7 +315,7 @@ open class MessageLabel: UILabel {
             switch result.resultType {
             case .address:
                 var ranges = rangesForDetectors[.address] ?? []
-                let tuple: (NSRange, MessageTextCheckingType) = (result.range, .addressComponents(result.addressComponents))
+                let tuple: (NSRange, MessageTextCheckingType) = (result.range, .addressComponents(result.components))
```

Comment:
Why did we change `.addressComponents` here?

Reply:
It has been marked as "// Deprecated in favor of components" so I updated it

Reply:
Looks good to me. I don't see this as deprecated in the Foundation docs. Just need to test that it works before merging 👍

Reply:
The comments not documented in `Apple API Reference`, so please make sure this is available in `iOS9` - `iOS11`.(IMO, maybe we can do this change after documented in `API Reference` 😄 )

Reply:
happy to take it out if you prefer, but prob a good one to remember for when it does go